### PR TITLE
add Flavio Finger to RECON members

### PR DESCRIPTION
**Pull request by zkamvar**
This adds @ffinger to the list of RECON members for his field deployment
work in DRC and his followup package contribution.

@ffinger, can you confirm that the details of this are correct?

### CODE REVIEW

**Comment by zkamvar** [comment]
Ping @ffinger

File: people.md [hunk 1]
```diff
@@ -124,6 +124,13 @@ members:
     url: http://www.imperial.ac.uk/people/zulma.cucunuba
     github: https://github.com/zulmac
     twitter: https://twitter.com/Zulma.Cucunuba
+  - name: Flavio Finger
+    img: /img/people/flavio-finger.jpg
+    desc: Research Fellow in real-time epidemiological analysis. London School of Hygene and Tropical Medicine, UK.
```
Comment (ffinger) [inline people.md]: This has changed since. Can you put "Epidemiologist & Outbreak Analyst at Epicentre, Paris, France"

**Review by ffinger — CHANGES_REQUESTED** [review]
Hi @zkamvar, sorry for the delay, hadn't seen this. Some things evolved. Would you mind removing the twitter, I am really not active there. No worries if that's complicated.

File: people.md (hunk 1 above)
Comment (zkamvar) [inline people.md]:
```suggestion
    desc: Epidemiologist & Outbreak Analyst at Epicentre, Paris, France
```

Comment (zkamvar) [reply people.md]: Done

File: people.md [hunk 2]
```diff
@@ -124,6 +124,13 @@ members:
     url: http://www.imperial.ac.uk/people/zulma.cucunuba
     github: https://github.com/zulmac
     twitter: https://twitter.com/Zulma.Cucunuba
+  - name: Flavio Finger
+    img: /img/people/flavio-finger.jpg
+    desc: Epidemiologist & Outbreak Analyst at Epicentre, Paris, France
+    url: https://ffinger.github.io
+    website: https://ffinger.github.io
+    twitter: https://twitter.com/flavio_finger
```
Comment (zkamvar) [inline people.md]:
```suggestion
```

**Comment by zkamvar** [comment]
> Hi @zkamvar, sorry for the delay, hadn't seen this. Some things evolved. Would you mind removing the twitter, I am really not active there. No worries if that's complicated.

Done

**Pull request merged**